index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" href="public/stylesheets/styles.css">
+
+  <title>Calculator Benchmark</title>
+
+</head>
+<body>
+  <div class="calculator-container">
+    <div>
+      <div class="calculator-body">
+        <div class="calculator-window">
+            <div class="btn-window btn-close btn-spacing">x</div>
+            <div class="btn-window btn-min btn-spacing">-</div>
+            <div class="btn-window btn-open btn-spacing">+</div>
+        </div>
+        <div class="calculator-nums">
+          <div class="calculator-row">
+            <div class="calculator-button calculator-num-colors">
+              <p class="calculator-font">AC</p>
```

Comment:
I'm not sure you need these extra internal elements inside of your button to get the styles you want. If you do I would definitely not use a paragraph tag here. These are not paragraphs. If you do need them use `<span>`. Another reason to avoid `<p>` tags here is because they come with margin that you definitely dont want here.